src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/NuGet.VisualStudio.Internal.Contracts.csproj

```diff
@@ -6,6 +6,7 @@
     <IncludeInVsix>true</IncludeInVsix>
     <RootNamespace>NuGet.VisualStudio.Internal.Contracts</RootNamespace>
     <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);CS8602</NoWarn>
```

Comment:
There are 66 errors and I didn't want to accidentally introduce more bugs, so I'll leave those changes for a future PR.

I'll make that changes in a new PR to avoid potentially breaking something sometihng.

Reply:
Issue: https://github.com/NuGet/Home/issues/13179